### DISCUSSION TITLE
Dúvida sobre uma informação na nota fiscal

### DISCUSSION THREAD

**Discussion by emillyhorta** [apyb/comunidade#146] [Geral]
Quero saber se tem como colocar uma informação a mais na NF, a ordem de compra. A empresa solicitou pra colocar.  @cecivieira

**Comment by naanadr** [comment]
Solicita a nota fiscal no fluxo oficial, e no email que chega para vocês informando da aberta da solicitação tu responde passando qual informação adicional você quer. Porque aí a @cecivieira fica ciente e temos o registro da solicitação.

**Reply by emillyhorta** [reply]
ah, okay. obrigada

**Reply by emillyhorta** [reply]
Feito!
